README.md

```
@@ -60,33 +62,67 @@ gitGraph
+# Non-interactive (CI):
+# SHIPLOG_BORING=1 shiplog write
+# or shiplog --boring write
 ```
 
-Use `SHIPLOG_ENV` to target a specific environment (defaults to `prod`). For convenience you can symlink the script to `shiplog` and invoke it directly.
+Use `SHIPLOG_ENV` to target a specific environment (defaults to `prod`).
+
+## Installation
+
+1. Clone the repository somewhere on your machine (for example, under `$HOME/.shiplog`):
+   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)